RELEASE.md

```diff
@@ -41,58 +41,70 @@ with `make upgrade-npm-deps`
 
 ### 1. Prepare your release
 
-You should start to create a branch that follows the pattern `release/v<X.Y>`. Release candidates and patch releases
-for any given major or minor release happen in the same `release/v<major>.<minor>` branch. Do not
-create `release/<version>` for patch or release candidate releases.
+#### Create a release branch
 
-- Do this in a proper PR pointing to the release branch as this gives others the opportunity to chime in on the release
-  in general and on the addition to the changelog in particular.
+Create a branch that follows the naming pattern `release/v<X.Y>` and includes the changes you intend to release (usually the latest from `main`). Push it to github. You will use this branch as the base in the next step.
 
-- Update the file `VERSION` with the new version to be created.
-
-- Update the file `CHANGELOG.md` :
-
-```bash
-make generate-changelog
-```
-
-Note: it will generate a first changelog version based on the git history. You should review what has been generated.
-Error can happen ;)
-
-- Entries in the `CHANGELOG.md` are meant to be in this order:
-
-* `[FEATURE]`
-* `[ENHANCEMENT]`
-* `[BUGFIX]`
-* `[BREAKINGCHANGE]`
-
-As we have many libraries we publish, it's better if you also put a clear indication about what library is affected by
-these changes.
+> ⚠️ Release candidates and patch releases for any given major or minor release happen in the same `release/v<major>.<minor>` branch. Do not create `release/<version>` for patch or release candidate releases.
```

Comment:
I actually don't understand what a release candidate is here (this already existed, but since you're nicely updating all this, I figure I'll share what's confusing me too) 

The release branch is `release/v<major>.<minor>`. If we have to patch it, this is saying the process is to create a branch and patch it into the release branch (right?)

That leaves me wondering, what is a release candidate / how is different from a release branch?

Reply:
Mainly because it says: `Do not create release/<version> for patch or release candidate releases.` Isn't the branch the release candidate?

Reply:
a release candidate is a release suffixed by `rc<number of the release candidate>`. For example `v0.28.0-rc0`
Making a release candidate or a release is the same process, it just change the tag itself. 

Let me know if it helps you to understand :) I hope it doesn't confuse you even more